client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -95,9 +99,15 @@ class JetpackOnboardingSummaryStep extends React.PureComponent {
 	}
 }
 
-export default connect( () => {
-	const tasks = compact( [] );
-	return {
-		tasks,
-	};
-} )( localize( JetpackOnboardingSummaryStep ) );
+export default connect( ( state, { siteId } ) => ( {
+	hasWooCommerceInstallFinished: get(
+		getRequest(
+			state,
+			saveJetpackOnboardingSettings( siteId, {
+				installWooCommerce: true,
+			} )
+		),
+		'hasLoaded'
+	),
+	tasks: compact( [] ),
```

Comment:
what is `compact( [] )` for?

Reply:
We're still working on this bit, and it's a preparation for supporting logic, defining which tasks to show. We'll probably need this as we start implementing each of the next tasks to display on this step.